docs/0001-strategy-for-maintaining-OS-versions.rst

```diff
@@ -1,62 +1,69 @@
 Title: Strategy for maintaining iOS versions in the OpenEdx Project
```

Comment:
Some formatting to improve readability on GitHub. I'll probably move into another PR.

Reply:
Thanks for improving the 0001 doc style 👍